vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue

```diff
@@ -200,10 +187,18 @@ export default {
       'Washington', 'West Virginia', 'Wisconsin', 'Wyoming',
     ],
   }),
+
+  computed: {
```

Comment:
Curious, why is this a computed property if it's just returning a boolean literal?